lib/commons/paragraph.dart

```diff
@@ -8,13 +8,14 @@ class Paragraph extends StatelessWidget {
   final String title;
   final Widget child;
   final double topPadding;
+  final double bottomPadding;
 
-  Paragraph({this.title, this.child, double topPadding}): this.topPadding = topPadding ?? 44.0;
+  Paragraph({this.title, this.child, this.topPadding = 20.0, this.bottomPadding = 20.0});
```

Comment:
`bottomPadding` with the value `0` seems to be the default no? Perhaps we can set the default to `0` to avoid setting this value everywhere?

Reply:
It's done !